ormconfig.ts

```diff
@@ -0,0 +1,17 @@
+export default {
+  type: 'postgres',
+  host: 'localhost',
+  port: 6000,
+  username: 'postgres',
+  driver: 'postgres',
+  password: 'idazov',
+  database: 'nodeJs',
+  entities: [__dirname + 'src/**/*.entity.{ts,js}'],
```

Comment:
use `path.join(__dirname, 'src/**/*.entity.{ts,js}')` for better path handling

Reply:
fixed